packages/core/getCalendarsEvents.ts

```diff
@@ -23,7 +23,7 @@ const getCalendarsEvents = async (
      * TODO: Migrate credential type or appId
      */
     const passedSelectedCalendars = selectedCalendars.filter((sc) => sc.integration === type);
-    if (!passedSelectedCalendars.length) return [];
+    if (!passedSelectedCalendars.length && type.indexOf("other_calendar") === -1) return [];
```

Comment:
This change allows to run `getAvailability` for calendar app credentials that are not considered a valid calendar per se, such as this app.